### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by elichad** [ResearchObject/ro-crate-2024#15]
Non-exhaustive list

- [x] links have very low contrast ratio (2.32:1), so they're hard to read - should be at least 4.5:1
- [x] links should be underlined or have another non-colour indication that they're links
- [x] some pages missing top-level heading (e.g. /use_cases and /faq), haven't checked them all
- [x] /use_cases - search form missing label
- [x] Skip links broken - they target `#main` but no page component has that ID
- [x] ~back to top button has `aria-hidden="true"`~ doesn't need fixing as SR users have other (more common) ways to get back to the start of the page. This comes direct from the ELIXIR theme anyway

(new 2024-05-03)
- [x] remove underline from Domain, Task, Role headings on homepage
- [ ] change Domain, Task, Role blocks to be same navigation style as the use cases, for consistency
